app_spec/test/regressions.js

```diff
@@ -118,11 +118,11 @@ scenario.runTape('create & publish post -> recommend to other agent', async (t,
     agent_address: tash.agentId
   })
   console.log("linked: ", linked)
-  t.equal(linked.Ok, "QmUFkn3kTXuFmBvynEMQ4ox3WyvkkG4GtfvPkNqb5b7Ge7")
+  t.ok(linked.Ok);
 
   const recommendedPosts = tash.call('blog', 'my_recommended_posts', {})
   console.log("recommendedPosts", recommendedPosts)
   console.log('agent addresses: ', alice.agentId, tash.agentId)
-
+  t.equal(recommendedPosts.Ok.links[0].address,"QmaERphTvXExCBcVdCRzGBE2d1dnRNWDjthwfyBuq9Mf6a");
```

Comment:
Can we keep these hashes out of the test? Aren't these going to change every time we update app spec? Is there a less brittle way we can do the same check?

Reply:
hmmm maybe find a way to generate them and then check against them?

Reply:
Is this really a useful test though? Would we really expect a linked address to be different from what we would expect? Wouldn't that mean holochain is broken? Does that check belong in an end-to-end test?

Reply:
This was an existing test just with a changed hash, we had a existing conversation with nico and I am probably going to remove some of the hashes because with the chain header it is not going to be deterministic anymore. 

Pros of links imo is the determinism because of the data where are putting in, we know what we are getting out, but if we already have coverage at the lower levels we can just check the number of links it gets back.

I don't really know why the regression.js is for to be honest because same setup + looks just like what we are doing on the app_spec side. Would be better if regression.js had the rust conductor